themes/stanford_basic/theme-settings.php

```diff
@@ -38,6 +38,18 @@ function stanford_basic_form_system_theme_settings_alter(array &$form, FormState
     '#default_value' => theme_get_setting('brand_bar_variant'),
   ];
 
+  // External Link toggle.
+  $form['options_settings']['ext_links_label'] = [
+    '#type' => 'fieldset',
+    '#title' => t('External Link Icon settings'),
+  ];
+
+  $form['options_settings']['ext_links_label']['hide_su_ext_links'] = [
```

Comment:
We should put this into the `Site Settings` config page and have the hook from the `.theme` file in a module. Perhaps `stanford_profile_styles` module. Then we won't have issues with sub themes.